cypress/integration/products/productsWithoutSku/productsWithoutSkuInOrder.js

```diff
@@ -1,30 +1,17 @@
 /// <reference types="cypress"/>
 /// <reference types="../../../support"/>
 
-import {
-  createCustomer,
-  deleteCustomersStartsWith
-} from "../../../support/api/requests/Customer";
+import { createCustomer } from "../../../support/api/requests/Customer";
 import { createReadyToFulfillOrder } from "../../../support/api/utils/ordersUtils";
-import {
-  createProductWithShipping,
-  deleteProductsStartsWith
-} from "../../../support/api/utils/products/productsUtils";
-import { deleteShippingStartsWith } from "../../../support/api/utils/shippingUtils";
+import { createProductWithShipping } from "../../../support/api/utils/products/productsUtils";
 import filterTests from "../../../support/filterTests";
 
-filterTests({ definedTags: ["all", "critical"] }, () => {
-  const name = "ProductsWithoutSkuInOrder";
+filterTests({ definedTags: ["all", "critical", "refactored"] }, () => {
+  const name =
```

Comment:
Shouldn't stay as it was? And also this name cannot be reused because it will fail at email validation cause it won't be unique. 
```suggestion
  const name = "ProductsWithoutSkuInOrder";
```